frontend/pages/shared-components/enrollment/call-to-action/call-to-action.module.css

```diff
@@ -0,0 +1,11 @@
+.ctaContainer {
```

Comment:
I think shared-components should go outside of pages directory. From what I read in the Nextjs docs all pages that are to be routable should go into pages directory, can we move the share-components to outside the pages directory?

Reply:
It is outside of the pages directory! Are you perhaps looking at an old version? I've squashed all my commits so I'm not sure why this displayed like that 
<img width="296" alt="Captura de pantalla 2021-07-08 a las 12 04 08 p  m" src="https://user-images.githubusercontent.com/50561643/124955419-08054500-dfe5-11eb-996d-42656020f15f.png">

Reply:
To clarify for other folks - turns out I introduced an accidental duplicate of the directory 🙈 will remove the one under `pages/` on next push!